config/rhoai/binding_admin_roles.yaml

```diff
@@ -0,0 +1,16 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kueue-admin-rolebinding
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: rhods-admins
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: odh-admins
```

Comment:
I vaguely remember @zdtsw mentioned there might be a third group?

Reply:
`dedicated-admins` for Managed cluster ?

Reply:
Please let me know if I have to add any other group to this file @astefanutti @zdtsw

Reply:
My understanding is that `dedicated-admins` should be added?

Reply:
thanks, `dedicated-admins` group added